docs/ChangeLog/20190830.md

```diff
@@ -7,7 +7,7 @@ This document marks the inaugural Breaking Change merge. A list of changes follo
 ## Core code formatting with clang-format
 
 * All core files (`drivers/`, `quantum/`, `tests/`, and `tmk_core/`) have been formatted with clang-format
-* A travis process to reformat PR's on merge has been instituted
+* A travis process to reformat PRs on merge has been instituted
```

Comment:
Travis? :D
```suggestion
* A GitHub Actions process to reformat PRs on merge has been instituted
```

Reply:
Nah, this is historical. We've got a GH Action, but it wasn't brought about at the time.